backend/src/src-vulnerabilities/src/main/java/org/eclipse/sw360/vulnerabilities/VulnerabilityHandler.java

```diff
@@ -12,6 +12,13 @@
  */
 package org.eclipse.sw360.vulnerabilities;
 
+import com.google.common.base.MoreObjects;
```

Comment:
all these changes probably warrant an update of the file header

Reply:
Done